### ISSUE TITLE
Auto-Login doesn't work with Joomla 3.4.3

### ISSUE THREAD

**Issue by roman1528** [arnecls/raidplaner#151]
Hi.
Bounded the Raidplaner 1.1.4 as iframe in my Joomla CMS.
Login to CMS, and then open the page with raidplaner result in a second login. 
Cockie settings are correct, raidplaner is in a sub directory off the CMS installation.
Login to raidplaner with joomla user-data works also fine.

what can i do to get this working?

Site: http://www.vanitate.de

Thank you very much.

**Comment by arnecls** [comment]
Did you have a look at this page yet? https://github.com/arnecls/raidplaner/wiki/Automatic-login-troubleshooting

**Comment by roman1528** [comment]
Yes I had. Domaincookie-setting is .vanitate.de cause of using subdomains and cookie-path is set to / cause raidplaner is installed into a subfolder of vanitate.de. 
i think raidplaner cannot found the session in db or get not the coockie.
I compared the sessionid's between cookie and joomla db with username... are correct.
maybe the iframe is the problem?

**Comment by arnecls** [comment]
Ok - normally this is either something with the JS Part (can be disabled by using cleartext transfer) or some detail changed with Joomla.
I will try to have a look at that at the weekend.
You will need to be a little bit patient at the moment because I've currently a lot to do at work (and because of this almost no free time).
